mritopng/__init__.py

```diff
@@ -92,3 +92,4 @@ def convert_folder(mri_folder, png_folder, auto_contrast=False):
                     print('SUCCESS: %s --> %s' % (mri_file_path, png_file_path))
                 except Exception as e:
                     print('FAIL: %s --> %s : %s' % (mri_file_path, png_file_path, e))
+                    os.remove(png_file_path)
```

Comment:
**Let's check if this file exists before removing it**

I feel like we're assuming here that this file will *always* get created even if there is an exception. There could be an exception in creating the file itself inside `convert_file(..)`

Reply:
Oh, right. My assumption was that since we are the ones listing all the files from folder, the source file will always exist. So the `Source file doesn't exist` will be taken care of. However I did forget about an exception being thrown in creating the file.